client/client.go

```diff
@@ -9,26 +9,24 @@ import (
 
 // Client sdkclient.Client and provides umee chain specific transactions and queries.
 type Client struct {
-	sdkclient.Client
+	*sdkclient.Client
```

Comment:
why do we need a pointer here? The embedded struct already has pointers. I don't think we need another level of pointers.